multiqc/utils/search_patterns.yaml

```
@@ -642,7 +642,7 @@ salmon/fld:
   fn: "flenDist.txt"
 sambamba/markdup:
   contents: "finding positions of the duplicate reads in the file"
-  num_lines: 5
+  num_lines: 10
```

This is a specific enough text and the speed difference between 5-50 lines is so small that I'm inclined to give it a bit more wiggle room, to give margin of error for the future..

```suggestion
  num_lines: 50
```